src/views/dashboards/Faas/AddRemoveStakeModal.vue

```diff
@@ -310,15 +328,24 @@ export default {
             "getUserOwnedNfts",
             this.stakingInfo.stakingTokenInfo.address
           );
-          // console.log(
-          //   "NFTTOKENS",
-          //   typeof this.allUserNftTokens,
-          //   Object.values(this.allUserNftTokens),
-          //   this.allUserNftTokens
-          // );
-          this.selectedNftTokenIds = Object.values(
-            this.allUserNftTokens
-          )[0].token_id;
+
+          // Filter NFT's that do not yet have metadata
+          this.allUserNftTokens = this.allUserNftTokens.filter((nft) => {
+            const metadata = JSON.parse(nft.metadata);
+            if (!metadata) return false;
+            return true;
+          });
```

Comment:
@whatl3y Sometimes I noticed it takes some time for the metadata to come across after minting. I thought we should just filter those out for now instead of showing empty NFT's since it might confuse the user. If we think we should show the NFT's (since the user might wonder where they are) but with empty placeholder images, we can do that as well.